x/auth/client/cli/tx_multisign.go

```diff
@@ -125,7 +125,8 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 			for _, sig := range sigs {
 				err = signing.VerifySignature(sig.PubKey, signingData, sig.Data, txCfg.SignModeHandler(), txBuilder.GetTx())
 				if err != nil {
-					return fmt.Errorf("couldn't verify signature: %w", err)
+					addr, _ := sdk.AccAddressFromHex(sig.PubKey.Address().String())
```

Comment:
Nit you can drop the `.String()` as the `%s` formatter will automatically do this.